docs/access.md

```diff
@@ -1,20 +1,118 @@
 # Access management
 
-## LaminDB
+This document outlines the access control system, detailing its architecture, key features, roles, and implementation guidelines.
 
-LaminDB doesn't come with access management but relies on the underlying data infrastructure for this.
+## System architecture
 
-## LaminHub
+Our access management is built on a four-tier hierarchy, providing granular control over resources:
 
-```{include} includes/access-laminhub.md
+1.  **Organizations:** The top-level entity. Contains all associated teams, instances, and spaces. Acts as the primary boundary for users and resources.
```

Comment:
I think this is somewhat unnecessary and confusing: The top-level entity.

I'd rather say: An organizational account can group users, instances, and other entities belonging to the same organization.